packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -329,16 +330,18 @@ export class MessageWidget extends Component {
    */
   @autobind
   listenToBufferState(conversationId, sparkInstance) {
```

Comment:
`conversationId` probably isn't needed any more since we are using the conversation via props